integration_tests/test_basic.py

```diff
@@ -598,3 +598,39 @@ def test_contract(cronos):
     # call contract
     greeter_call_result = contract.caller.greet()
     assert "world" == greeter_call_result
+
+
+def test_tx_inclusion(cronos):
+    """
+    - send multiple heavy transactions at the same time.
+    - check they are included in consecutively blocks without failure.
+    """
+    w3 = cronos.w3
+    # bigger than block_gas_limit/2, so at most one tx in a block
+    tx_gas_limit = 80000000
+    amount = 1000
+    # use different sender accounts to be able be send concurrently
+    signed_txs = []
+    for account in ["validator", "community", "signer1", "signer2"]:
+        signed_txs.append(
+            sign_transaction(
+                w3,
+                {
+                    "to": ADDRS["validator"],
```

Comment:
the actual gas to send one transaction is 21,000


From your test I understand that the anteHandler will count the gas_limit, not the actual gas used. Isnt it a problem?